scopesim/tests/tests_integrations/test_micado_cluster.py

```diff
@@ -1,100 +0,0 @@
-"""Simulate a large cluster with ScopeSIM.
```

Comment:
Why is this test removed? It is kinda a regression test that ensures problems like https://github.com/AstarVienna/ScopeSim/issues/111 cannot occur again undetected. Can we put the test back?